openhands/linter/linter.py

```diff
@@ -0,0 +1,35 @@
+import os
+from collections import defaultdict
+
+from openhands.linter.base import BaseLinter, LinterException, LintResult
+from openhands.linter.languages.python import PythonLinter
+from openhands.linter.languages.treesitter import TreesitterBasicLinter
+
+
+class Linter(BaseLinter):
```

Comment:
I kinda feel like a better name for this might be `DefaultLinter` or something? `Linter` makes me feel like it is a base class.

Reply:
I had the same feeling as well, i'll rename it!

Reply:
done in most recent commit!